lib/ui/semantics.dart

```diff
@@ -8,10 +8,12 @@ part of dart.ui;
 
 /// The possible actions that can be conveyed from the operating system
 /// accessibility APIs to a semantics node.
-//
-// When changes are made to this class, the equivalent APIs in
-// `lib/ui/semantics/semantics_node.h` and in each of the embedders *must* be
-// updated.
+///
+/// \warning When changes are made to this class, the equivalent APIs in
+///         `lib/ui/semantics/semantics_node.h` and in each of the embedders
+///         *must* be updated.
+/// See also:
+///   - file://./../../lib/ui/semantics/semantics_node.h
```

Comment:
looks like this didn't do what we hoped:

https://master-api.flutter.dev/flutter/dart-ui/SemanticsAction-class.html

Reply:
filed https://github.com/flutter/flutter/issues/94494

Reply:
A couple of years later... https://github.com/flutter/engine/pull/52229